app/ui/browser/views/navbar/navbar.jsx

```diff
@@ -229,4 +230,9 @@ NavBar.propTypes = {
   navigateTo: PropTypes.func.isRequired,
 };
 
-export default NavBar;
+const mapStateToProps = (state) => ({
+  // @TODO get this from the restructured reducer
```

Comment:
In general, TODO comments that don't explain what happens now, what should happen, and why the one is better than the other, don't add much.